server/sheet.go

```diff
@@ -337,6 +344,105 @@ func (s *Server) registerSheetRoutes(g *echo.Group) {
 		return nil
 	})
 
+	g.GET("/sheet/shared", func(c echo.Context) error {
+		ctx := c.Request().Context()
+		sheetFind := &api.SheetFind{}
+
+		if projectIDStr := c.QueryParam("projectId"); projectIDStr != "" {
```

Comment:
We can probably exact some common logic into a helper function such as composing partial Find{}.